index.html

```diff
@@ -7,8 +7,26 @@
     <title></title>
 </head>
 <body>
-    <div class="main-container" >
+     <section class="section-bio" >
          
-    </div>
+          <div class="main-container" >
+            <div class="bio-header"><p>BIO</p></div>
+            
+                    <div class="bio-box ">
```

Comment:
I recommend using the BEM convention in naming the classes like : 
`bio__header` , `bio__box`
like what the mentor @YakoobHammouri has said.